src/08_comprehensions.py

```diff
@@ -11,13 +11,17 @@
 # Write a list comprehension to produce the array [1, 2, 3, 4, 5]
 
 y = []
+for i in range(5):
+    y.append(i+1)
 
 print (y)
 
 # Write a list comprehension to produce the cubes of the numbers 0-9:
 # [0, 1, 8, 27, 64, 125, 216, 343, 512, 729]
 
 y = []
+for i in range(10):
```

Comment:
You used a function to loop through and create the array here. You can do this all together in a list comprehension `y = [x**3 for x in range(10)]`